portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTestWithoutACConnection.testcase

```diff
@@ -70,6 +71,77 @@ definition {
 		}
 	}
 
+	@description = "LPS-108147: When user opens AB testing panel and AC is connected but not synced, the AB testing panel displays instructions to sync to AC ('Don't show me this again' button and the message is not displayed)"
+	@priority = "3"
+	test CheckABTestPanelInfoNotSyncAC {
+		task ("Open Analytics Cloud") {
+			var analyticsCloudURL = PropsUtil.get("analytics.cloud.url");
+
+			Navigator.openURL(baseURL = "${analyticsCloudURL}");
+		}
+
+		task ("Take token from Analytics Cloud") {
+			AnalyticsCloud.loginAnalyticsCloud(
+				emailAddress = "test@liferay.com",
+				password = "test");
+
+			AnalyticsCloud.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Go to DXP portal") {
+			Navigator.openURL();
+
+			User.loginPG(
+				password = "test",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Connect, but not sync site with Analytics Cloud") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(locator1 = "AnalyticsCloudConnection#ANALYTICS_CLOUD_DXP_BUTTON");
+
+			Click(locator1 = "AnalyticsCloudConnection#DXP_TOKEN_INPUT");
+
+			Paste(locator1 = "AnalyticsCloudConnection#DXP_TOKEN_INPUT");
+
+			Click(locator1 = "AnalyticsCloudConnection#CONNECT_ANALYTICS_CLOUD_BUTTON");
+
+			AssertTextEquals(
+				locator1 = "AnalyticsCloudConnection#ANALYTICS_CLOUD_CONNECTED_CONFIRMATION_MESSAGE",
+				value1 = "Your DXP instance is connected to Analytics Cloud.");
+		}
```

Comment:
Add a new site that is not synced to AC by default

Reply:
great idea, thanks. I moved to the other file, as this new way the connection with the AC can be done normally.